int32 group_id
geometry_msgs/Pose ring_pose
string color